### DISCUSSION TITLE
Cant get it working

### DISCUSSION THREAD

**Discussion by tomascalta** [andyrozman/ggc#5] [General]
Hi, tried to install latest and second latest version (0.7.0 and 0.6.0) and none of the version works. CMD just closes after split second after starting the program. Moving files from lib/hibernate3 doesnt work. What do I do? I really wanna try this out

**Comment by andyrozman** [comment]
I got pulled away on other project and I couldn't finalize my changes and make release, but if you follow steps in Building (on readme file), you should be able to get it running... All versions under 0.8.x are outdated, but theoretically if you download full package it should work... 
Where are you trying to run it? 
For windows it should install it, if you download full image (see ggc.sourceforge.net webpage), it should run also on linux or mac, but you would need to do it from commandline: 
Just unpack the files and go into bin folder and run appropriate batch command... It should work even on windows... If you are on windows you might have some permission issues... This was tested with much older versions where permissions where not as strict as they are now...

**Reply by tomascalta** [reply]
I got it working now. Thx.
